.github/workflows/ci_cmake.yml

```diff
@@ -0,0 +1,135 @@
+# CI workflow for wxWidgets builds using CMake.
+name: CMake builds
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '.github/workflows/ci_msw_cross.yml'
```

Comment:
Shouldn't we ignore `ci.yml` too here or is there some reason to not do it?

Reply:
Yes, and I'll ignore this script in `ci.yml`.